yaml/_type/omap.ts

```diff
@@ -17,18 +15,18 @@ function resolveYamlOmap(data: Any): boolean {
   for (const pair of data) {
     pairHasKey = false;
 
-    if (_toString.call(pair) !== "[object Object]") return false;
+    if (!isObject(pair)) return false;
```

Comment:
This isn't equivalent change. `isObject` is true for arrays and functions, but the existing code is not